src/Files.App/Helpers/EnvHelpers.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Files.Backend.Enums;
+using Microsoft.Extensions.Hosting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.ApplicationModel;
+
+namespace Files.App.Helpers
+{
+	public static class EnvHelpers
+	{
+	
+		public static (AppEnvironment, string) GetAppEnvironmentAndLogo()
+		{
+
+
+			var env =
+#if STORE
+				AppEnvironment.WindowsStore;
+#elif PREVIEW
+				AppEnvironment.WindowsSideloadPreview;
+#elif STABLE
+				AppEnvironment.WindowsSideload;
+#elif DEBUG
+				AppEnvironment.Debug;
+#else 
+				AppEnvironment.WindowsDev;
+#endif
+
+			var path = env switch
+			{
+				AppEnvironment.Release => Constants.AssetPaths.DevLogo,
```

Comment:
```suggestion
				AppEnvironment.Dev=> Constants.AssetPaths.DevLogo,
```

Reply:
This would mean changing the EnvironmentName to Dev, because it's set as the string representation of the enum variant. Do we want to do that?

Reply:
You're right, can you replace the `release` and `debug` enum with `Dev`.

Reply:
Ok